src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -204,3 +205,16 @@
             [:wallet/trigger-share-collectible
              {:title title
               :uri   uri}]]]})))
+
+(rf/reg-event-fx
+ :wallet/navigate-to-opensea
+ (fn [db [chain-id token-id contract-address]]
+   {:fx [[:dispatch [:hide-bottom-sheet]]
+         [:dispatch
+          [:browser.ui/open-url
+           (collectible-utils/get-opensea-collectible-url
+            {:chain-id               chain-id
+             :token-id               token-id
+             :contract-address       contract-address
+             :test-networks-enabled? (get-in db [:profile/profile :test-networks-enabled?])
+             :is-goerli-enabled?     (get-in db [:profile/profile :is-goerli-enabled?])})]]]}))
```

Comment:
Be careful, that `db` received in the `fn` is a `cofx` map, you should destructure it (e.g. `{db :db}`)

Reply:
I think if we test it with a testnet collectible, this event won't return the correct link.

Should this event be tested?

Reply:
well I have a test for the util that get's the url. 
I could move it to instead test this out at the event level if that suits better. 

I mostly wanted to lock in that nobody changes some constant names etc and misses it breaking the url

Reply:
I think the test for the utils function is enough :+1: